### ISSUE TITLE
Does not work when using template literals

### ISSUE THREAD

**Issue by danwenzel** [vuejs/eslint-plugin-vue#1271]
**Tell us about your environment**
* **ESLint version: tried with 6.8.0 and 7.6.0** 
* **eslint-plugin-vue version: tried with 6.2.2 and 7.0.0-beta.1** 
* **Node version: 12.17.0** 

**Please show your full configuration:**

```js
module.exports = {
  extends: [
    'plugin:vue/recommended'
  ]
}
```

The plugin is completely ignoring any html code inside of a template literal, like here:

```js
import Vue from 'vue';

Vue.component('MyCheckbox', {
  data() {
    return { checked: false, title: 'Check me' }
  },
  methods: {
    check() { this.checked = !this.checked; }
  },
  template: `
    <div class="checkbox-wrapper" @click="check">
      <div v-if="currentPage === 'index'">
      </div>
      <div v-else-if="currentPage === 'index'">
        duplicate v-else-if... should fail
      </div>
    </div>
  `,
});
```

Is there a way to get it to work with these?

**Comment by ota-meshi** [comment]
Thank you for this issue.

This plugin does not support string templates. Please use the `<template>` in the `.vue` file or do not use this plugin.
